### ISSUE TITLE
Support for Door close sensor in August Pro lock

### ISSUE THREAD

**Issue by bradleyscott** [snjoetw/py-august#2]
The August Smart lock Pro comes with a door open / close sensor. At the moment the status Enum is limited to Locked and Unlocked states, not Locked and Open, etc etc. 

I'll have a look at some point around what the API returns and maybe make a pull request if I can figure it out. But thought I'd capture it here in the first instance

**Comment by sfiorini** [comment]
I have two August Smart Locks, the normal ones, not pro (http://august.com/products/august-smart-lock/) and they come with "Door Sense" which is that additional sensor to install on the door which determines the open and close. I did run this API in debug mode and indeed found the extra attribute on the lock status. It is called "doorState". This is what I got: {"status":"unlocked","dateTime":"2018-04-12T15:26:49.881Z","isLockStatusChanged":false,"valid":true,"doorState":"closed"}
So, yeah, it's just a matter of returning an additional property on the lock status and then modify the home assistant component to "read that also".
